### PR TITLE
Multiple improvements and corrections

### PR DESCRIPTION
**Pull request by zeromido**
Improved options.html see 307ad4a
Improved options.js see 462b80d
Added New General Settings screenshot see f843d1a
@pfn Thanks for your advices :) I'm waiting your feedback!

### CODE REVIEW

File: chromeipass/options/options.html [hunk 1]
```diff
@@ -118,7 +117,7 @@ <h3 class="muted"><img src="/icons/keepass_48x48.png" alt="logo" /> chromeIPass<
 					</div>
 				</div><!-- /.navbar -->
 			</div>
-
+
```
Comment (pfn) [inline chromeipass/options/options.html]: delete space

File: chromeipass/options/options.html [hunk 2]
```diff
@@ -270,7 +279,7 @@ <h3 class="muted"><img src="/icons/keepass_48x48.png" alt="logo" /> chromeIPass<
 					</p>
 				</div>
 			</div>
-
+
```
Comment (pfn) [inline chromeipass/options/options.html]: delete space

File: chromeipass/options/options.html [hunk 3]
```diff
@@ -349,7 +357,7 @@ <h3 id="myModalLabel">Remove identifier from Chrome?</h3>
 					</div>
 				</div>
 			</div>
-
+
```
Comment (pfn) [inline chromeipass/options/options.html]: delete space

File: chromeipass/options/options.html [hunk 4]
```diff
@@ -394,7 +401,7 @@ <h3 id="myModalLabel">Remove specified credential fields?</h3>
...
```
Comment (pfn) [inline chromeipass/options/options.html]: delete space

File: chromeipass/options/options.html [hunk 5]
```diff
@@ -415,13 +425,10 @@ <h3 id="myModalLabel">Remove specified credential fields?</h3>
 					<div class="span6"><em class="versionKPH"></em></div>
 				</div>
 			</div>
-
 			<hr>
-
 			<div class="footer">
-				<p>2013 - Perry Nguyen, Lukas Schulze</p>
+				<p>2010-2016 - Perry Nguyen, Lukas Schulze</p>
```
Comment (pfn) [inline chromeipass/options/options.html]: delete

File: chromeipass/options/options.js [hunk 6]
```diff
@@ -69,12 +68,11 @@ options.initGeneralSettings = function() {
...
 
-
 	$("#showDangerousSettings").click(function() {
-		$("#dangerousSettings").show();
-		$(this).hide();
+        $('#dangerousSettings').is(":visible") ? $(this).text("Show these settings anyway") : $(this).text("Hide");
+        $("#dangerousSettings").toggle();
 	});
-
+
```
Comment (pfn) [inline chromeipass/options/options.js]: delete space

File: chromeipass/options/options.js [hunk 7]
```diff
@@ -120,48 +118,52 @@ options.initGeneralSettings = function() {
...
-		saveBlinkTimeoutSetting();
-	});
-	function saveBlinkTimeoutSetting() {
-		saveIntSetting("blinkTimeout", $("#blinkTimeout").val());
-	}
+		$("#blinkTimeoutButton").click(function(){
+		var blinkTimeout = $.trim($("#blinkTimeout").val());
+		var blinkTimeoutval = parseInt(blinkTimeout);
+		
+        options.settings["blinkTimeout"] = blinkTimeoutval.toString();
```
Comment (pfn) [inline chromeipass/options/options.js]: fix indent

File: chromeipass/options/options.js [hunk 8]
```diff
@@ -120,48 +118,52 @@ options.initGeneralSettings = function() {
...
 		localStorage.settings = JSON.stringify(options.settings);
+
 		chrome.extension.sendMessage({
 			action: 'load_settings'
 		});
-	}
+	});
 };
-
+
```
Comment (pfn) [inline chromeipass/options/options.js]: delete space

**Comment by pfn** [comment]
fix the minor things and we're good, thank you

**Comment by zeromido** [comment]
Well, I think it's better that titles aren't clickable, and the label tag makes title clickable (hand shaped cursor). What do you think about it?

**Comment by zeromido** [comment]
It's just from my browsing experience most of the titles like that ain't clickable
https://en.wikipedia.org/wiki/Wikipedia#History isn't clickable 
Anyway it won't make a big difference
Also, why anchor links don't work like "options.html#connected-databases"
I'm always redirected to "options.html#general-settings" which is the main page
any idea

**Comment by pfn** [comment]
I don't know why the anchor links don't work. And for semantics, since we
don't have any other <label> I'd prefer that the title might as well be the
label. It makes focusing the text fields easier.

**Comment by zeromido** [comment]
Whatever you say it's your project and the final word will be yours

**Pull request closed without merging**